dockerfile

```
@@ -1,12 +1,63 @@
-FROM python:3.8-alpine
+FROM python:3-slim
 
-RUN apk add zlib-dev jpeg-dev gcc musl-dev xvfb
+ENV POETRY_HOME="/opt/poetry" \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_NO_INTERACTION=1 \
+    GECKODRIVER="0.35.0"
 
-RUN pip install --upgrade pip;
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+ENV ENV="dev"
+
+RUN apt-get update && apt-get install -y \
+    gcc \
+    musl-dev \
+    xvfb\
```

Is this duplicated on line 41?
also nit
```suggestion
    xvfb \
```